src/citrine/resources/dataset.py

```diff
@@ -268,7 +290,7 @@ def delete(self, uid: Union[UUID, str, LinkByUID, DataConcepts], *, dry_run=Fals
             collection = self.gemd._collection_for(uid)
         else:
             collection = self.gemd
-        return collection.delete(uid, dry_run=dry_run)
+        return collection.delete(uid=uid, dry_run=dry_run)
```

Comment:
Are we intending to obsolete deletion with just an object ID? What's the point of this change?

Reply:
Same as above.

Reply:
Same as above: the `delete` signature hasn't changed, just the invocation here.